controls/radgridview/export/how-to/export-header.md

```diff
@@ -5,7 +5,7 @@ description: Export Custom Header
 slug: gridview-export-troubleshooting-export-header
 tags: export,custom,header
 published: True
-position: 1
+position: 3
 ---
 
 # Export Custom Header
```

Comment:
Intro of the article is confusing - is the problem that headers are used or their export?
Consider making this an ordered list and include each snippet in its respective list item.